tomcat8/hooks.win/init

```diff
@@ -0,0 +1,32 @@
+Write-Host "Preparing TOMCAT_HOME..."
+
+Write-Host "sys.ip        = {{sys.ip}}"
```

Comment:
We shouldn't be echoing these lines out.  IIRC this is a holdover from early days when people were learning and experimenting, but we should tighten up core plans.

Reply:
no problem will clean those up. left them in there for testing and just never took them out.